vnext/Scripts/TestServers.ps1

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+#
+# TestServers.ps1
+#
+function Start-Npm {
```

Comment:
> Start-Npm [](start = 8, length = 10)

It feels that it could be better to use Python or Perl language for scripts as they are supported on other platforms like Mac. Otherwise we will need to rewrite similar code for Mac in the future.

Reply:
We do not:

1. PowerShell is better designed for starting processes and commands.
1. PowerShell is available on Mac (and Linux) for a long while now.
1. This project is Windows-only, so it's not likely we will need to port this logic anyway (but if we ever do, see previous point).